README.md

```diff
@@ -11,6 +11,8 @@ https://docs.rs/remote-trait-object)
 
 A simple and powerful Rust remote method invocation library based on Rust trait objects.
 
+See the [documentation](https://docs.rs/remote-trait-object).
+
```

Comment:
I think giving a link of a simple example or embedding the example in the README.md file helps.
When I first met a library, I check an example in the README.md file before reading details.

Reply:
I updated